phoebe/atmospheres/passbands.py

```diff
@@ -39,7 +39,7 @@
 
 # define the URL to query for online passbands.  See tables.phoebe-project.org
 # repo for the source-code of the server
-_url_tables_server = 'http://tables.phoebe-project.org'
+_url_tables_server = os.getenv('PHOEBE_TABLES_SERVER', 'http://tables.phoebe-project.org')
```

Comment:
let's replace http with https while we're doing this

Reply:
there are a lot of other instances, I'll do that across the board in a follow-up PR